.clang-tidy

```diff
@@ -11,9 +12,12 @@ Checks:
     cert-env33-c,
     cert-err34-c,
     cert-err*-cpp,
+    -cert-err58-cpp,
     cert-flp30-c,
     cert-oop*,
     cppcoreguidelines-*,
+    -cppcoreguidelines-avoid-magic-numbers,
+    -cppcoreguidelines-pro-bounds-array-to-pointer-decay,
```

Comment:
I am in favor of demoting this check to a warning at the very least, but I am unsure about disabling it entirely. Thoughts?

Reply:
That's acceptable. Will demote to warning.

Reply:
Done.